### ISSUE TITLE
tag field is missing form label

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#938]
<img width="351" alt="screen shot 2016-10-27 at 15 38 36" src="https://cloud.githubusercontent.com/assets/13200338/19771648/89f517f2-9c5b-11e6-960d-0dae12af7a1e.png">
